samples/package.json

```diff
@@ -13,7 +13,9 @@
   "dependencies": {
     "@google-cloud/speech": "^2.3.1",
     "@google-cloud/storage": "^2.0.0",
+    "chalk": "^2.4.2",
     "node-record-lpcm16": "^0.3.0",
+    "stream": "0.0.2",
```

Comment:
This isn't necessary, `stream` is part of the Node.js stdlib and can be `require`-d without adding as a dependency.